dynamic_rest/prefetch.py

```diff
@@ -244,6 +244,14 @@ def execute(self):
         use_models = has_fastquery and self.model.use_fastquery == False
 
         if use_models:
+            prefetches = [
+                Prefetch(
+                    prefetch.field,
+                    queryset=prefetch.query.queryset
+                ) for prefetch in self.prefetches.values()
+            ]
+            if len(prefetches) > 0:
+                qs = qs.prefetch_related(*prefetches)
```

Comment:
This makes all downstream prefetches/sideloads also slow, but sounds like that's deliberate for now.